autorest/codegen/models/__init__.py

```diff
@@ -155,7 +157,14 @@ def build_type(yaml_data: Dict[str, Any], code_model: CodeModel) -> BaseType:
         code_model.types_map[yaml_id] = response
         response.fill_instance_from_yaml(yaml_data, code_model)
     else:
-        response = TYPE_TO_OBJECT[yaml_data["type"]].from_yaml(yaml_data, code_model)  # type: ignore
+        if yaml_data["type"] in TYPE_TO_OBJECT:
+            response = TYPE_TO_OBJECT[yaml_data["type"]].from_yaml(yaml_data, code_model)  # type: ignore
```

Comment:
this code can be simplified, since i'm resolving merge conflicts, I did the code simplification. Let me know your thoughts!

Reply:
Thanks @iscai-msft !